tests/subsys/mgmt/zcbor_bulk/src/main.c

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <ztest.h>
+#include <sys/byteorder.h>
+#include <zcbor_common.h>
+#include <zcbor_decode.h>
+#include <zcbor_bulk/zcbor_bulk_priv.h>
+
+
+void test_correct(void)
+{
+	/* { "hello":"world", "one":1, "bool_value":true } */
```

Comment:
```suggestion
	/* { "hello":"world", "one":1, "bool_val":true } */
```
I think this is what is found below.

Reply:
Yeah, will fix.